src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,270 @@
+            printHorizontalLine();
+        }
+    }
+
+    private static void addTodo(ArrayList<Task> tasks, String line) {
+        String todoDescription = line.replaceFirst("todo ", "");
+        Todo newTodo = new Todo(todoDescription);
+        tasks.add(newTodo);
+        int todoId = tasks.size() - 1;
+        printNewTask(tasks.get(todoId), tasks.size());
+    }
+
+    static void tryUnmarkTask(ArrayList<Task> tasks, String line) {
+        try {
+            if (line.replaceFirst("unmark", "").trim().equals("")) {
+                throw new DukeException();
+            }
+            unmarkTask(tasks, line.split(" "));
+        } catch (DukeException e) {
+            printHorizontalLine();
+            System.out.println("     T_T OOPS!!! The description of an unmark cannot be empty.");
+            printHorizontalLine();
+        }
+    }
+
+    private static void unmarkTask(ArrayList<Task> tasks, String[] parsedInput) {
+        int unmarkId = Integer.parseInt(parsedInput[1]) - 1;
+        tasks.get(unmarkId).setNotDone();
+        printUnmark(tasks.get(unmarkId));
+    }
+
+    static void tryMarkTask(ArrayList<Task> tasks, String line) {
+        try {
+            if (line.replaceFirst("mark", "").trim().equals("")) {
+                throw new DukeException();
+            }
+            markTask(tasks, line.split(" "));
+        } catch (DukeException e) {
+            printHorizontalLine();
+            System.out.println("     T_T OOPS!!! The description of a mark cannot be empty.");
+            printHorizontalLine();
+        }
+    }
+
+    private static void markTask(ArrayList<Task> tasks, String[] parsedInput) {
+        int markId = Integer.parseInt(parsedInput[1]) - 1;
+        tasks.get(markId).setDone();
+        printMark(tasks.get(markId));
+    }
+
+    private static void deleteTask(ArrayList<Task> tasks, int deleteId) {
+        printHorizontalLine();
+        System.out.println("     Noted. I've removed this task:");
+        System.out.println("     " + tasks.get(deleteId));
+        System.out.println("     Now you have " + (tasks.size() - 1) + " tasks in the list.");
+        printHorizontalLine();
+        tasks.remove(deleteId);
+    }
+
+    static void tryDeleteTask(ArrayList<Task> tasks, String[] parsedInput) {
+        int deleteId = Integer.parseInt(parsedInput[1]) - 1;
+        try {
+            if (deleteId > tasks.size()) {
+                throw new DukeException();
+            }
+            deleteTask(tasks, deleteId);
+        } catch (DukeException e) {
+            printHorizontalLine();
+            System.out.println("     T_T OOPS!!! There is no such task number");
+            printHorizontalLine();
+        }
+    }
+
+    public static void readFile(String filePath, ArrayList<Task> tasks) throws FileNotFoundException {
+        File f = new File(filePath); // create a File for the given file path
```

Comment:
According to Java coding standard, comments should be indented relative to their position in code.